src/client/pps/pps.proto

```diff
@@ -25,9 +25,20 @@ enum JobState {
     JOB_STATE_SUCCESS = 2;
 }
 
+enum Partition {
+  BLOCK = 0;
+  FILE = 1;
+  REPO = 2;
+}
```

Comment:
Protobuf compiles this to contants like `Partition_BLOCK`.  If we rename it to `PARTITION_BLOCK`, it'd become `Partition_PARTITION_BLOCK`, which seems unwieldy.